packages/pocket-ponics-app/screens/tier-screen.js

```diff
@@ -105,8 +127,8 @@ export default class Example extends React.Component {
 						</Text>
 					</View>
 				</View>
-				<TouchableOpacity>
-					<Text style={styles.button}>Harvest {this.getReadableName(plant.name)}</Text>
+				<TouchableOpacity style={styles.button} onPress={() => this.props.navigation.navigate('HarvestInstruction', {name: this.getReadableName(plant.name), })}>
+					<Text style={styles.buttonText}>Harvest {this.getReadableName(plant.name)}</Text>
```

Comment:
Make sure this button only displays when the plant is actually ready to harvest - see the Seedling screen for an example